README.md

```diff
@@ -18,22 +19,44 @@ Here's a demo GIF to give you a glimpse of a user interacting with the webapp an
 Make sure you have Python version 3.11 or higher installed.
 
 Install the dependencies.
+
 ```
-pip3 install -r .devcontainer/requirements
+pip3 install -r .devcontainer/requirements.txt
 ```
 
 In ubuntu, install the packages.
 ```
 sudo xargs -a .devcontainer/packages.txt apt-get install -y
 ```
 
+**Remark:** You can set the directory path in the PYTHONPATH environment variable so that the imports of other files from the repository are working properly.
+In a conda environment enter (do not forget to edit the /path/to/add):
+
+```
+conda env config vars set PYTHONPATH=$PYTHONPATH:/path/to/add
+```
+
+After a restart of the environment :
+
+```
+conda deactivate
+conda activate your_env_name
+```
+
+You can check that the environment variable has been successfully updated:
+
+```
+conda env config vars list
+```
+
+You should see the updated PYTHONPATH value.
```

Comment:
please could you also mention that on linux you can alternatively add this line in your `.bashrc` file so that each new terminal adds the current path to the PYTHONPATH 
`export PYTHONPATH="$PYTHONPATH:$PWD"`

Reply:
Done